t/documentation.t

```diff
@@ -0,0 +1,30 @@
+use Test;
+use Data::Dump::Tree;
+use Agrammon::DataSource::CSV;
+use Agrammon::Documentation;
+use Agrammon::Model;
+use Agrammon::Model::Parameters;
+use Agrammon::OutputFormatter::CSV;
+use Agrammon::TechnicalParser;
+
+my $model-version = 'hr-inclNOxExtendedWithFilters';
+my $path = $*PROGRAM.parent.add("test-data/Models/$model-version/");
+my $model = Agrammon::Model.new(:$path);
+lives-ok { $model.load('Total') }, "Could load module Total from $path";
+#lives-ok { $model.load('Livestock::DairyCow') }, "Could load module Livestock::DairyCow from $path";
+
+my $params;
+lives-ok
+    { $params = parse-technical($*PROGRAM.parent.add("test-data/Models/$model-version/technical.cfg").slurp) },
+    'Parsed technical file';
+isa-ok $params, Agrammon::Model::Parameters, 'Correct type for technical data';
+
+lives-ok { prepare-model(
+               $model,
+               technical => %($params.technical.map(-> %module {
+                   %module.keys[0] => %(%module.values[0].map({ .name => .value }))
+               }))
+           );
+}, 'Successfully created docu';
```

Comment:
I wonder if, for a more robust test, you could feel the LaTeX output to - assuming there is one on path - a latex program, that would at least assert that it is valid LaTeX syntax.

Reply:
I'd like to post-pone this a bit, as I will need to add a `pdflatex` command anyway for the (not yet implemented) PDF report generation. Once that is implemented, that additional test will be trivial.
Added a TODO, though.

Reply:
OK, works for me.